sources/packages/web/src/components/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -69,12 +69,8 @@ export default defineComponent({
       {} as StudentApplicationOfferingChangeRequestAPIInDTO,
     );
     const approveApplicationOfferingChangeRequest = ref({} as VForm);
-    const formModel = reactive(
-      {} as StudentApplicationOfferingChangeRequestAPIInDTO,
-    );
     const cancel = () => {
-      approveApplicationOfferingChangeRequest.value.reset();
-      formModel.studentConsent = false;
+      approveApplicationOfferingChangeRequestModal.value.studentConsent = false;
```

Comment:
I have a point, to mention, but not a blocker. do we need a model for just one property coming from form ? As the code is part of previous PR not stressing it.

Reply:
I used a model to keep it in sync with how we are doing it in other places.